_includes/hero.html

```diff
@@ -1,6 +1,6 @@
 <div class="hero">
   <div class="container">
     <img class="logo" src="{{ site.baseurl }}/img/logo.png" alt="Istio">
-    <h1 class="hero-lead">A service mesh for polyglot microservices</h1>
+    <h1 class="hero-lead">An open platform to connect, manage, and secure microservices</h1>
```

Comment:
I had a similar thought, but I'm worried if we drop the micro it doesn't make it as clear that we operate at the infrastructure layer. Can I leave as microservices for now and we debate that separately?